src/Sylius/Behat/Context/Ui/CartContext.php

```diff
@@ -13,8 +13,10 @@
 
 use Behat\Behat\Context\Context;
 use Sylius\Behat\Page\Cart\CartSummaryPage;
```

Comment:
Since we use interfaces the specific implementations are not used any more and should be removed from import.